docs/react.md

```diff
@@ -94,15 +94,26 @@ const messagePreviews = messages.map((msg, index) => <li key={index}>{msg.data.s
 `useChannel` supports all of the parameter combinations of a regular call to `channel.subscribe`, so you can filter the messages you subscribe to by providing a `message type` to the `useChannel` function:
 
 ```javascript
-const { channel } = useChannel("your-channel-name", "test-message", (message) => {
+useChannel("your-channel-name", "test-message", (message) => {
     console.log(message); // Only logs messages sent using the `test-message` message type
 });
 ```
 
+#### useChannel `publish` function
+
+The `publish` function returned by `useChannel` can be used to send messages to the channel.
+
+```javascript
+const { publish } = useChannel("your-channel-name")
+publish("test-message", { text: "message text" });
+```
+
+#### useChannel `channel instance`
+
 The `channel` instance returned by `useChannel` can be used to send messages to the channel. It's just a regular Ably JavaScript SDK `channel` instance.
 
 ```javascript
-channel.publish("test-message", { text: "message text" });
+const { channel } = useChannel("your-channel-name")
```

Comment:
the actual publish call is now missing. basically, keep the old and the new line, like this:
```jsx
const { channel } = useChannel("your-channel-name")
channel.publish("test-message", { text: "message text" });
```